sematic/api/endpoints/runs.py

```diff
@@ -100,16 +107,42 @@ def list_runs_endpoint(user: Optional[User]) -> flask.Response:
     after_cursor_count : int
         Number of items remain after the current cursor, i.e. including the current page.
     content: List[Run]
-        A list of run JSON payloads. The size of the list is `limit` or less if
-        current page is last page.
+        A list of run JSON payloads, if the 'include' request parameter was not set.
+        If the 'include' parameter was set to ['id'], returns a list of run ids. The
```

Comment:
Can we return a list of dictionaries instead to be consistent?
```python
content=[{"id": "foo"}, {"id": "bar"}, ...]
```
essentially a subset of the full objet dictionary.

This way when we eventually support partial object queries across the board we are already compliant.